src/QueryBuilder.php

```diff
@@ -84,19 +84,30 @@ public function withUseAclSpecification(UseAclSpecification $useAclSpecification
         return $this;
     }
 
+    private const QUERY_STRING_REPLACEMENTS = [
+        '"' => '',
+        '\'' => '',
+        '\\' => '\\\\'
+    ];
+
     public static function create(): self
     {
         return new self();
     }
 
     public function getSearchParams(string $queryString, string $type, int $start, int $count, string $order, string $dir): array
     {
-        $queryString = str_replace(['"', '\''], '', $queryString);
+        $queryString = str_replace(
```

Comment:
I just wanted to make sure that, if you type ",\,' we still can find resources containing this as part of the data. Example: 

- If I have an item called `'Item'` will it work?
- If I have an item called `\Item` will it work?
- If I have an item called `"Item"` will it work?

Reply:
Ok, lets postpone it and revisit on the partial search, where this type of character will be required. 

Thanks for the explanation